### PR TITLE
[zh-cn]: create the translation of `documentpipevent.windows` property

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
part of #23557

### CODE REVIEW

File: files/zh-cn/web/api/documentpictureinpictureevent/window/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+slug: Web/API/DocumentPictureInPictureEvent/window
+l10n:
+  sourceCommit: 89c435da452257b944b403cc9e45036fcb22590e
+---
+
+{{APIRef("Document Picture-in-Picture API")}}{{SeeCompatTable}}{{SecureContext_Header}}
+
+## `window` 属性
+
+**`window`** 只读属性属于 {{domxref("DocumentPictureInPictureEvent")}} 接口，返回一个 {{domxref("Window")}} 实例，表示事件触发时 `DocumentPictureInPicture` 窗口中的浏览上下文。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/documentpictureinpictureevent/window/index.md:12]:
```suggestion
{{domxref("DocumentPictureInPictureEvent")}} 接口的 **`window`** 只读属性，返回一个 {{domxref("Window")}} 实例，表示事件触发时 `DocumentPictureInPicture` 窗口中的浏览上下文。
```

File: files/zh-cn/web/api/documentpictureinpictureevent/window/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+title: DocumentPictureInPictureEvent：window 属性
+slug: Web/API/DocumentPictureInPictureEvent/window
+l10n:
+  sourceCommit: 89c435da452257b944b403cc9e45036fcb22590e
+---
+
+{{APIRef("Document Picture-in-Picture API")}}{{SeeCompatTable}}{{SecureContext_Header}}
+
+## `window` 属性
+
```
Comment (jasonren0403) [inline files/zh-cn/web/api/documentpictureinpictureevent/window/index.md:11]:
```suggestion
```

File: files/zh-cn/web/api/documentpictureinpictureevent/window/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+{{domxref("DocumentPictureInPictureEvent")}} 接口的 **`window`** 只读属性，返回一个 {{domxref("Window")}} 实例，表示事件触发时 `DocumentPictureInPicture` 窗口中的浏览上下文。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/documentpictureinpictureevent/window/index.md:12]:
```suggestion
{{domxref("DocumentPictureInPictureEvent")}} 接口的 **`window`** 只读属性返回一个 {{domxref("Window")}} 实例，表示事件触发时 `DocumentPictureInPicture` 窗口中的浏览上下文。
```

**Review by jasonren0403 — COMMENTED** [review]
minor changes

**Comment by skyclouds2001** [comment]
wonder why this is closed @tianyeeT

**Pull request closed without merging**